content/docs/k8s/gateway-api.mdx

```diff
@@ -2,15 +2,17 @@
 title: Gateway API
 sidebar_position: 3
 lang: en-US
-keywords: [kubernetes, k8s, gateway api]
+keywords: [kubernetes, k8s, gateway api, ingress, ingress controller]
 ---
 
-Starting in v0.28, the Pomerium Ingress Controller includes experiment support for the Kubernetes [Gateway API].
+The [Pomerium Ingress Controller](https://github.com/pomerium/ingress-controller) includes experimental support for the Kubernetes [Gateway API].
 
 ## Overview
 
 Gateway API is an alternative to the Ingress API, aimed at standardizing more of the common functionality between different ingress implementations, and with a focus on role-based resource management.
 
+If you are already using the Gateway API to model service networking in Kubernetes or are interested in using it for the first time with Pomerium, you can use the Kubernetes Gateway API with the Pomerium Ingress Controller to manage supported ingress routes.
```

Comment:
Thanks for the suggestion. I've tweaked this to ask for feedback to either support@pomerium.com or https://discuss.pomerium.com.